### DISCUSSION TITLE
Insufficient Privilege

### DISCUSSION THREAD

**Discussion by teshiburu2020** [koush/scrypted#846] [Q&A]
Heya 

I'm trying to set this up with the unifi protect plugin, but I keep getting this error 

[UniFi Protect] Insufficient privileges for this user. Please check the roles assigned to this user and ensure it has sufficient privileges.

I've tried with both the user having full access to everything and just read only to cameras with the same fault in all I adjust

**Comment by galligan** [comment]
Also having this issue now today! Same thing.
